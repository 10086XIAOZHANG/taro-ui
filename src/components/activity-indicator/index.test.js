import Nerv from 'nervjs'
import { renderToString } from 'nerv-server'

import AtActivityIndicator from '../../../.temp/components/activity-indicator/index'

describe('ActivityIndicator Snap', () => {
  it('render initial ActivityIndicator', () => {
    const componet = renderToString(<AtActivityIndicator />)
    expect(componet).toMatchSnapshot()
  })

  it('render ActivityIndicator -- props size', () => {
    const componet = renderToString(<AtActivityIndicator size={32} />)
    expect(componet).toMatchSnapshot()
  })

  it('render ActivityIndicator -- props color', () => {
    const componet = renderToString(<AtActivityIndicator color='#13CE66' />)
    expect(componet).toMatchSnapshot()
  })

  it('render ActivityIndicator -- props content ', () => {
    const componet = renderToString(<AtActivityIndicator content='加载中...' />)
    expect(componet).toMatchSnapshot()
  })

  it('render ActivityIndicator -- props mode ', () => {
    const componet = renderToString(<AtActivityIndicator mode='center' />)
    expect(componet).toMatchSnapshot()
  })
})
