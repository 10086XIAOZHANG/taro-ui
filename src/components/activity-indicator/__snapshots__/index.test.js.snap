// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityIndicator Snap render ActivityIndicator -- props color 1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#13CE66;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`ActivityIndicator Snap render ActivityIndicator -- props content  1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div><span class=\\"taro-text at-activity-indicator__content\\">加载中...</span></div>"`;

exports[`ActivityIndicator Snap render ActivityIndicator -- props mode  1`] = `"<div class=\\"at-activity-indicator at-activity-indicator--center\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`ActivityIndicator Snap render ActivityIndicator -- props size 1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:32px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;

exports[`ActivityIndicator Snap render initial ActivityIndicator 1`] = `"<div class=\\"at-activity-indicator\\"><div class=\\"at-activity-indicator__body\\"><span style=\\"font-size:24px;color:#6190E8;\\" class=\\"taro-text at-icon at-icon-loading\\"></span></div>undefined</div>"`;
